data/json/items/comestibles/carnivore.json

```
@@ -345,6 +345,7 @@
     "type": "COMESTIBLE",
     "name": "meat jerky",
     "name_plural": "meat jerky",
+    "contextual_names": [ [ "COMPONENT_ID", "mutant", "monster jerky" ] ],
```

I think these would be clearer as objects, something like
```suggestion
    "contextual_names": [ { "COMPONENT_ID": "mutant", name: "monster jerky" } ],
```
or
This would probably be preferred, but is longer.
```suggestion
    "contextual_names": [ { "type": "COMPONENT_ID", "value": "mutant", "name": "monster jerky" } ],
```
See https://github.com/CleverRaven/Cataclysm-DDA/pull/34602#issuecomment-541276015.
I hope this wouldn't make it much more complex.